cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -78,27 +85,29 @@ executions {
     }
 }
 
-def config = new ConfigSlurper().parse(new File(ctx.artifactoryHome.haAwareEtcDir, PROPERTIES_FILE_PATH).toURL())
-log.info "Schedule job policy list: $config.policies"
-
 config.policies.each{ policySettings ->
     def cron = policySettings[ 0 ] ? policySettings[ 0 ] as String : ["0 0 5 ? * 1"]
     def repos = policySettings[ 1 ] ? policySettings[ 1 ] as String[] : ["__none__"]
     def months = policySettings[ 2 ] ? policySettings[ 2 ] as int : 6
     def paceTimeMS = policySettings[ 3 ] ? policySettings[ 3 ] as int : 0
     def dryRun = policySettings[ 4 ] ? policySettings[ 4 ] as Boolean : false
     def disablePropertiesSupport = policySettings[ 5 ] ? policySettings[ 5 ] as Boolean : false
+    def keepRelease = policySettings[ 6 ] ? policySettings[ 6 ] as Boolean : false
+    def releaseRegex = policySettings[ 7 ] ? policySettings[ 7 ] as Pattern : ~/.*-\d+\.\d+\.\d+\.*/
+
+    log.info "Schedule job policy list: $config.policies"
+    log.info "Schedule regex: $releaseRegex"
 
     jobs {
         "scheduledCleanup_$cron"(cron: cron) {
-            log.info "Policy settings for scheduled run at($cron): repo list($repos), months($months), paceTimeMS($paceTimeMS) dryrun($dryRun) disablePropertiesSupport($disablePropertiesSupport)"
-            artifactCleanup( months, repos, log, paceTimeMS, dryRun, disablePropertiesSupport )
+            log.info "Policy settings for scheduled run at($cron): repo list($repos), months($months), paceTimeMS($paceTimeMS) dryrun($dryRun) disablePropertiesSupport($disablePropertiesSupport) keepRelease($keepRelease), releaseRegex($releaseRegex)"
+            artifactCleanup( months, repos, log, paceTimeMS, dryRun, disablePropertiesSupport, keepRelease, releaseRegex )
         }
     }
 }
 
-private def artifactCleanup(int months, String[] repos, log, paceTimeMS, dryRun = false, disablePropertiesSupport = false) {
-    log.info "Starting artifact cleanup for repositories $repos, until $months months ago with pacing interval $paceTimeMS ms, dryrun: $dryRun, disablePropertiesSupport: $disablePropertiesSupport"
+private def artifactCleanup(int months, String[] repos, log, paceTimeMS, dryRun = false, disablePropertiesSupport = false, keepRelease = false, releaseRegex = ~/.*-\d+\.\d+\.\d+\.*/) {
```

Comment:
Why is the default value for keepRelease set to false? Wouldn't it be safer to set it to true to avoid accidental removal?